### ISSUE TITLE
How to adjust the learning rate with a single GPU training

### ISSUE THREAD

**Issue by 237014845** [ZJULearning/ttfnet#2]
How to adjust the learning rate with a single GPU training

**Comment by liuzili97** [comment]
Hi, the learning rate will automatically adjust even if you change the number of Gpus. Please check the `--autoscale-lr` in `tools/train.py`.

However, if you change the batch size per gpu (i.e., `imgs_per_gpu` in the config file), you need to adjust the learning rate on your own. For example, if the `imgs_per_gpu` changes from 16 to 8, the learning rate should be halved.

**Comment by 237014845** [comment]
@liuzili97 You use 8 GPU training, batchsize=128, lr=0.016, I use a single GPU training, batchsize=16, then the learning rate is adjusted to lr=0.002 right?

**Comment by liuzili97** [comment]
@237014845 No. The batch size is composed of `gpu_num * imgs_per_gpu`. If the `imgs_per_gpu` is not changed, the learning rate is not supposed to be modified.

Therefore, the lr should be 0.016 here.

**Comment by 237014845** [comment]
@liuzili97 ok, thank you
